docs/general/how-to/use-token-creator.md

```diff
@@ -0,0 +1,138 @@
+| **Symbol\***            | Short identifier (e.g., MCT, FAN).                                   |
+| **Category**            | The token type of the asset ([more details](#2-choosing-a-category)) |
+| **Initial Mint Amount** | Initial amount of the created asset.                                 |
+
+:::warning \*Token name and Symbol
+Note that the `Name` and the `Symbol` values can not be changed later.
+:::
+
+![Metadata entry form](../../../static/img/general/how-to/token-creator/token-details.png)
+
+### 2. Choosing a Category
+
+Select what your token represents. Examples include:
+
+- Fan Token
+- Proof of Attendance
+- Utility Token
+- Collectible
+- Membership
+
+The system automatically pre-selects the appropriate token type (LSP7 or LSP8), but you can change it if needed.
+
+![Category selection dropdown](../../../static/img/general/how-to/token-creator/tokentype-cheatsheet.jpg)
+
+### 3. Review and Create!
+
+Review the entered information and click **Create**.
+![Transaction confirmation screen](../../../static/img/general/how-to/token-creator/confirm-token-details.png)
+
+Sign the transactions from your Universal Profile to confirm deployment.
+![Transaction confirmation screen](../../../static/img/general/how-to/token-creator/transactions-list.png)
+
+After confirmation, the token will appear under your profile.
+![Transaction confirmation screen](../../../static/img/general/how-to/token-creator/token-on-profile.png)
+
+## Managing Tokens
+
+After creation, you can manage your token through your asset page on **Universal Everything**.
+
+Simply click the **Edit** icon to start managing your tokens:
+
+![Manage token screen](../../../static/img/general/how-to/token-creator/edit-button-highlight.png)
+
+### Edit Token Metadata
+
+When edit mode is enabled, it is possible to add more metadata to your token such as **image**, **description**, **external links**, and **assets** of your token.
+
+![Manage token screen](../../../static/img/general/how-to/token-creator/save-token-metadata.png)
+
+Once you are done with entering details, click **Save** icon to add these details to your token.
+
+### Mint & Burn Tokens
+
+**a. Mint More Tokens**
+
+- It is possible to mint more tokens when **Edit** mode is enabled:
+
+![Manage token screen](../../../static/img/general/how-to/token-creator/mint-more.png)
+
+- You can either mint it to your Universal Profile, or to an another UP by searching it's name!
+
+![Manage token screen](../../../static/img/general/how-to/token-creator/mint-popup.png)
+
+**b. Burn Tokens**
+
+- It is possible to burn the tokens you created on the token page directly without switching to Edit mode.
+
+![Manage token screen](../../../static/img/general/how-to/token-creator/burn-token.png)
+
+:::warning Zero Address
+
+Burning the tokens would send them to **zero address(0x....dead)**, meaning that they are irrecoverable once sent.
```

Comment:
```suggestion
Burning the tokens would send them to **zero address(0x....dead)**, meaning that they are irrecoverable once sent. This will also reduce the token total supply.
```